categorysite.html

```diff
@@ -66,117 +66,116 @@ <h4>Varukorg</h4>
 			</div>
 		</nav>
 <!-- header end -->
-    <div class="container col-12 col-md-6 flex-column">
-    	<div class="row">
+    <div class="container col-sm-12 col-md-12 d-flex flex-nowrap">
+    	<div class="col-md-7">
     	<content class="content">
 			<section class="Matkassar">
 				<div class="col-12 col-md-12"> 
```

Comment:
Om ni vill att alla kategorier skall ta upp alla 12 kolumner, för alla skärmstorlekar, då kommer kategorierna vara i en kolumn hela tiden.

Minns dock att så var ej fallet för er första webbsida.
testa sätta ändra col-md-12 til col-md-6, för alla kategorier.
Då blir det åtminstone 2 kategorier på rad, vid brytpunkten `md`


```suggestion
				<div class="col-12 col-md-6"> 
```